presto-plugin-toolkit/src/test/resources/table.json

```diff
@@ -1,4 +1,18 @@
 {
+  "schemas": [
+    {
+      "user": "user_with_access_to_bobschema",
+      "schema": "bobschema",
+      "owner": false,
+      "allow": "read-only"
+    },
```

Comment:
please also add a rule with `allow: true`

Reply:
In ` AccessMode::fromJson `, if value is true, it return ` ALL `. I didn't made the change, it's in previous commit

Reply:
I would rather not have a test with `allow: true`. The fact we're supporting it is to simplify code only.
(We support `allow: true` in the other place for backward compatibility. Here there is no backward compatibility in play.)
Having such a test would suggest we commit ourselves to support this.

Reply:
sure will do